Card/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     testImplementation 'org.skyscreamer:jsonassert:1.4.0'
     testImplementation project(':TestUtils')
     testImplementation project(':ThreeDSecure')
-    testImplementation "androidx.core:core-ktx:+"
+    testImplementation "androidx.core:core-ktx:1.6.0"
```

Comment:
Updating the `core-ktx` dependencies to specific versions is unrelated to this PR, but using `+` is causing tests to fail, and we should be pinning to a specific version.